openfpga_flow/benchmarks/quicklogic_tests/and2/and2.act

```diff
@@ -0,0 +1,3 @@
+a 0.5 0.5
+b 0.5 0.5
```

Comment:
@lnsharma Can you remove these duplicated micro benchmarks?

Reply:
Removed